scripts/benchmark.py

```diff
@@ -59,7 +60,8 @@ def main(
             print(f.read())
         print()
 
-        try:
+        # surrpess exception
```

Comment:
typo: `surrpess` -> `suppress`

Reply:
Also, not very helpful comment I would remove altogether tbh

Reply:
I added the dummy comment on the old PR to test only the CI workflow